dqgen/services/additions_query_generator.py

```diff
@@ -6,25 +6,82 @@
-        # output = template_builder.build(template,p1,p2,p3)
-        # return output
-        pass
+    def __init__(self, cls: str, operation, output_folder_path: str):
+        self.cls = cls
+        self.operation = operation
+        self.output_folder_path = output_folder_path
+
+    def build_file_path(self):
+        return query_file_name_builder.make_query_file_name(output_folder_path=self.output_folder_path,
+                                                            operation=self.operation,
+                                                            cls=self.cls)
+
+    def build_query_template(self):
+        template = QueryTemplateRegistry().INSTANCE_ADDITIONS
+        output = template_builder.build_template(jinja2_template=template, cls=self.cls)
+
+        return output
 
 
 class SimplePropertyAdditionsGenerator(BaseQueryGenerator):
     """
-        TBD
+        This class will build the property additions query as a file and
+        will put the file into the defined output folder
+    """
+
+    def __init__(self, cls: str, property: str, operation, output_folder_path: str):
+        self.cls = cls
+        self.operation = operation
+        self.output_folder_path = output_folder_path
+        self.property = property
+
+    def build_file_path(self):
+        return query_file_name_builder.make_query_file_name(output_folder_path=self.output_folder_path,
+                                                            operation=self.operation,
+                                                            cls=self.cls,
+                                                            prop=self.property)
+
+    def build_query_template(self):
+        template = QueryTemplateRegistry().PROPERTY_ADDITIONS
+        output = template_builder.build_template(jinja2_template=template, cls=self.cls, prop=self.property)
+
+        return output
+
+
+class ReifiedPropertyAdditionsGenerator(BaseQueryGenerator):
+    """
+        This class will build the reified property additions query as a file and
+        will put the file into the defined output folder
     """
 
-    def build_query(self) -> str:
-        pass
+    def __init__(self, cls: str,property: str, object_property: str, operation, output_folder_path: str):
+        self.cls = cls
+        self.operation = operation
+        self.output_folder_path = output_folder_path
+        self.property = property
+        self.object_property = object_property
+
+    def build_file_path(self):
+        return query_file_name_builder.make_query_file_name(output_folder_path=self.output_folder_path,
+                                                            operation=self.operation,
+                                                            cls=self.cls,
+                                                            prop=self.property,
+                                                            obj_prop=self.object_property)
+
+    def build_query_template(self):
```

Comment:
Building query template, as mentioned above, 
this could move up to the parent class, and have the template be provided as a parameter.